examples/catTables.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 from __future__ import print_function
+from past.builtins import basestring
```

Comment:
On python3 they are synonyms. In Python2 I think the reason I do this is because string literals are a different class to the override `str` and I needed `basestring` to cover all possible variants.